pkg/operator/ceph/csi/csi.go

```diff
@@ -67,7 +67,7 @@ func (r *ReconcileCSI) setParams() error {
 		return errors.Wrap(err, "unable to parse value for 'ROOK_CSI_ENABLE_NFS'")
 	}
 
-	if CSIParam.EnableCSIHostNetwork, err = strconv.ParseBool(k8sutil.GetOperatorSetting("CSI_ENABLE_HOST_NETWORK", "true")); err != nil {
+	if CSIParam.EnableCSIHostNetwork, err = strconv.ParseBool(k8sutil.GetOperatorSetting("CSI_ENABLE_HOST_NETWORK", "false")); err != nil {
```

Comment:
How does this impact the old csi driver? I'm afraid the old csi driver will break if we change this default.

Reply:
Your questions is for upgrade clusters? Or users who want to just revert back to old driver

Reply:
For any users who want/need to revert back to the old driver

Reply:
For users who want to revert they will manually need to set it `true` as now it default is false

Reply:
After a bit more digging and input from @BlaineEXE, we realized the old csi provisioner only enabled host networking when `ROOK_ENFORCE_HOST_NETWORK: true`. This setting for `CSI_ENABLE_HOST_NETWORK` was not ever applied to the provisioner. So we should revert this change to the default since it doesn't apply to the provisioner.